source/jormungandr/tests/routing_tests.py

```diff
@@ -99,6 +99,12 @@ def test_shared_section(self):
         """
         pass
 
+    def test_journeys_too_short_heavy_mode_fallback_filter(self):
+        """
+        This new feature is not supported in default
```

Comment:
This may conflict with PR removing default and destineo's scenario : #2616 
No idea what's the best to deal with that.

Reply:
First come, first serve ;) And this is more critical than destineo's removal !